config_defaults.yml

```diff
@@ -44,6 +44,9 @@ appveyor.yml:
 MAINTAINERS.md:
   maintainers:
     - "Puppet Forge Modules Team `forge-modules |at| puppet |dot| com`"
+NOTICE:
+  # copyright-begin: '2011'   # Beginning date of copyright (*) Please consult legal as to what these dates ACTUALLY mean and when they should be changed
+  # copyright-end: '2015'     # End date of copyright (*) Please consult legal as to what these dates ACTUALLY mean and when they should be changed
```

Comment:
This also doesn't handle multiple groups with copyright. I think you could go the simpler route of just making this text (and a list).